dotcom-rendering/src/web/components/EmailSignup.tsx

```diff
@@ -18,15 +18,25 @@ type Props = {
 const containerStyles = css`
 	border: ${neutral[0]} 3px dashed;
 	border-radius: 12px;
-	padding: 10px;
 	margin-bottom: ${space[3]}px;
+	padding: ${space[2]}px;
+
+	${from.tablet} {
+		padding: ${space[2]}px ${space[3]}px;
+		& > div {
+			margin-bottom: -4px;
+		}
```

Comment:
We try to avoid these types of selectors in DCR, they create couplings between components that can make it hard to reason about later. Can we target the css using local css instead.

That said, negative margins are a bit of a flag - is there a better way to achieve the spacing we want?

Reply:
There was some odd behaviour  in the `NewsletterFrequency` component that was making it taller than it should be, but I've made a change to that so we don't need the div or the negative margin.